src/modules/dashboard/components/ExpenditurePage/Stages/StakeExpenditureDialog.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useCallback } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import Dialog, { DialogSection } from '~core/Dialog';
+import { Form, Toggle } from '~core/Fields';
+import Heading from '~core/Heading';
+import Icon from '~core/Icon';
+import Numeral from '~core/Numeral';
+import { Tooltip } from '~core/Popover';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+import Button from '~core/Button';
+import styles from './StakeExpenditureDialog.css';
+
+// Mock Data for Staking token, needs to be replaced with native token.
+const activeToken = {
+  address: '0x0000000000000000000000000000000000000000',
+  balances: [{ amount: '0', domainId: 0 }],
+  decimals: 18,
+  iconHash: '',
+  id: '0x0000000000000000000000000000000000000000',
+  name: 'Ether',
+  symbol: 'ETH',
+};
+
+const MSG = defineMessages({
+  header: {
+    id: 'dashboard.Expenditures.Stages.draftConfirmDialog.header',
+    defaultMessage: 'Stake to Create Expenditure',
+  },
+  force: {
+    id: 'dashboard.Expenditures.Stages.draftConfirmDialog.force',
+    defaultMessage: 'Force',
+  },
+  stake: {
+    id: 'dashboard.Expenditures.Stages.draftConfirmDialog.stake',
+    defaultMessage: 'Stake',
+  },
+  descriptionText: {
+    id: 'dashboard.Expenditures.Stages.draftConfirmDialog.descriptionText',
+    defaultMessage: `Almost there! You have to provide a stake first. 
+    Imagine it is like renting a permission. 
+    If motion succeeds you will get your stake back.`,
+  },
+  descriptionText2: {
+    id: 'dashboard.Expenditures.Stages.draftConfirmDialog.descriptionText2',
+    defaultMessage: `
+    Until expenditure is staked it won’t show up publicly on list. 
+    It works like an anti-spam filter.`,
+  },
+  cancelText: {
+    id: 'dashboard.Expenditures.Stages.draftConfirmDialog.cancelText',
+    defaultMessage: 'Cancel',
+  },
+  confirmText: {
+    id: 'dashboard.Expenditures.Stages.draftConfirmDialog.confirmText',
+    defaultMessage: 'Stake',
+  },
+});
+
+interface Props {
+  cancel: () => void;
+  close: () => void;
+  onClick?: () => void;
+}
+
+const StakeExpenditureDialog = ({ cancel, onClick, close }: Props) => {
+  const handleSubmit = useCallback(() => {
+    onClick?.();
+    close();
+  }, [onClick, close]);
+
+  return (
+    <Dialog cancel={cancel}>
+      <div className={styles.dialogContainer}>
```

Comment:
I think that this is an unnecessary div element. It only adds the side padding. Instead you can do the theme `sidePadding`. This will be consistent with all other dialogs (You can check `MintTokens` dialog to see an example). I just think it's good to have consistency across all of the dialogs.
Also we have all of our dialogs in a separate folder. I think that it's better to keep them there unless there is a real reason not to.